packages/opentelemetry-cloud-monitoring-exporter/src/transform.ts

```diff
@@ -119,6 +121,35 @@ export function createTimeSeries(
   };
 }
 
+function transformResource(
+  resource: Resource,
+  projectId: string
+): { type: string; labels: { [key: string]: string } } {
+  const cloudProvider = `${resource.labels['cloud.provider']}`;
+  // These are the only supported resources
+  if (cloudProvider === 'gcp') {
+    return {
+      type: 'gce_instance',
+      labels: {
+        instance_id: `${resource.labels['host.id']}`,
+        project_id: projectId,
+        zone: `${resource.labels['cloud.zone']}`,
```

Comment:
I would suggest to use constants defined in OT JS instead of direct values, to avoid breaking change in future. https://github.com/open-telemetry/opentelemetry-js/blob/abd191bef366de8926f14701bf8b83df03b60d22/packages/opentelemetry-resources/src/constants.ts#L34